src/convertToObject.js

```diff
@@ -6,7 +6,27 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const resultObject = {};
```

Comment:
Avoid using generic names like `resultObject` for variables. Use descriptive names that reflect the purpose of the variable.

Reply:
> Avoid using generic names like `resultObject` for variables. Use descriptive names that reflect the purpose of the variable.

Good point